.github/workflows/run-help.yml

```diff
@@ -0,0 +1,121 @@
+---
+name: run-help
+'on': [push, pull_request]
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+jobs:
+  help-linux:
+    name: help-linux
+    runs-on: ubuntu-latest
+    #container: ghcr.io/lmms/linux.gcc:20.04
+    steps:
+      - name: wait for build
+        uses: lewagon/wait-on-check-action@v1.3.4
+        with:
+          ref: ${{ github.event.pull_request.head.sha || github.sha }}
+          check-name: 'linux'
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          wait-interval: 30
+      - name: Download workflow artifact
+        uses: dawidd6/action-download-artifact@v6
+        with:
+          workflow: build.yml
+          name: linux
+      - name: Extract artifact
+        run: |
+          chmod +x lmms-*.AppImage
+          ./lmms-*.AppImage --appimage-extract
+      - name: Display help
+        run: |
+          cd squashfs-root/
+          ./AppRun --help | grep "Usage: lmms"
+  help-macos:
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - arch: 'x86_64'
+            runner: 'macos-latest'
+          - arch: 'arm64'
+            runner: 'macos-14-arm64'
+    name: help-macos-${{ matrix.config.arch }}
+    runs-on: ${{ matrix.config.runner }}
+    steps:
+      - name: wait for build
+        uses: lewagon/wait-on-check-action@v1.3.4
+        with:
+          ref: ${{ github.event.pull_request.head.sha || github.sha }}
+          check-name: macos-${{ matrix.config.arch }}
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          wait-interval: 30
+      - name: Download workflow artifact
+        uses: dawidd6/action-download-artifact@v6
+        with:
+          workflow: build.yml
+          name: macos-${{ matrix.config.arch }}
+      - name: Mount the DMG
+        run: |
+          mkdir -p ~/tmpvolume
```

Comment:
Apple seems to create the directory if it's missing, so `mkdir -p` should be safe to remove.